.eslintrc.js

```diff
@@ -1325,6 +1325,14 @@ module.exports = {
         'packages/kbn-config-schema',
         'src/plugins/status_page/**',
         'src/plugins/saved_objects_management/**',
+        'packages/kbn-analytics/**',
```

Comment:
It's time we updated the comment https://github.com/elastic/kibana/pull/93086/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R1319